app/classifier/tasks/shortcut.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { Markdown } from 'markdownz';
+
+class Summary extends React.Component {
+  render() {
+    let answer;
+
+    if (this.props.annotation.value) {
+      answer = (
+        <div className="answer">
+          <i className="fa fa-check-circle-o fa-fw" />
+          <Markdown tag="span" inline={true}>{this.props.task.answers[this.props.annotation.value].label}</Markdown>
+        </div>
+      );
+    } else {
+      answer = <div className="answer">No answer</div>;
```

Comment:
In the dev classifier as well as in a test project, when I click the shortcut option and click done, the summary I see is 'No answer'